ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+    </rule>
+
+    <rule id="91557" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^29$</field>
+        <description>Office 365: Submission events from Exchange Online Protection and Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>MailSubmission</group>
+    </rule>
+
+    <rule id="91558" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^30$</field>
+        <description>Office 365: Microsoft Power Automate (formerly called Microsoft Flow) events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftFlow</group>
+    </rule>
+
+    <rule id="91559" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^31$</field>
+        <description>Office 365: Advanced eDiscovery events.</description>
+        <options>no_full_log</options>
+        <group>AeD</group>
+    </rule>
+
+    <rule id="91560" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^32$</field>
+        <description>Office 365: Microsoft Stream events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftStream</group>
+    </rule>
+
+    <rule id="91561" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^33$</field>
+        <description>Office 365: Events related to DLP classification in SharePoint.</description>
+        <options>no_full_log</options>
+        <group>ComplianceDLPSharePointClassification</group>
+    </rule>
+
+    <rule id="91562" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^34$</field>
+        <description>Office 365: Campaign-related events from Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>ThreatFinder</group>
+    </rule>
+
+    <rule id="91563" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^35$</field>
+        <description>Office 365: Microsoft Project events.</description>
+        <options>no_full_log</options>
+        <group>Project</group>
+    </rule>
+
+    <rule id="91564" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^36$</field>
+        <description>Office 365: SharePoint List events.</description>
+        <options>no_full_log</options>
+        <group>SharePointListOperation</group>
+    </rule>
+
+    <rule id="91565" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^37$</field>
+        <description>Office 365: SharePoint comment events.</description>
+        <options>no_full_log</options>
+        <group>SharePointCommentOperation</group>
+    </rule>
+
+    <rule id="91566" level="3">
```

Comment:
Security and Compliance Center events, could it be level 5?

Reply:
Done